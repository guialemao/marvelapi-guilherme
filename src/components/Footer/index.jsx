import React from 'react';
import styled from 'styled-components';

const FooterComponent = styled.footer`
  background-color: #fc1711;
  height: 70px;
  width: 100%;
`;

const Footer = () => <FooterComponent />;

export default Footer;
