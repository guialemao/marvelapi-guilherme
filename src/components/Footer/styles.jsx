import styled from 'styled-components';

const FooterComponent = styled.footer`
  background-color: #fc1711;
  height: 70px;
  width: 100%;
`;

export default FooterComponent;
