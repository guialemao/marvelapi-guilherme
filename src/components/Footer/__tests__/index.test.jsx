import React from 'react';
import { render, cleanup } from '@testing-library/react';

import Footer from '../index';

afterEach(cleanup);

describe('Testing Footer component', () => {
  it('should create a snapshot', () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment(<Footer />)).toMatchSnapshot();
  });
});
