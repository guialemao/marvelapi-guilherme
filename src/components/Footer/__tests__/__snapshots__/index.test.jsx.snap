// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer component should create a snapshot 1`] = `
<DocumentFragment>
  <footer
    class="sc-bdnylx jXptSh"
  />
</DocumentFragment>
`;
